src/app/kbv/controllers/self-assessment-tax-return-question.js

```diff
@@ -27,20 +27,26 @@ class SelfAssessmentTaxReturnQuestionController extends BaseController {
       }
 
       try {
-        const userInput = JSON.stringify({
-          statePension: req.body.statePension || req.body.statePensionShort,
-          otherPension: req.body.otherPension || req.body.otherPensionShort,
-          employmentAndSupportAllowance:
-            req.body.employmentAndSupportAllowance ||
+        const userInput = [
```

Comment:
With this, we probably still want to keep it as a key-value object so we can see what each value represents if wanted?

We can still reduce but would look more like:
```javascript
const totalPensionContribution = Object.keys(pensionContributions).reduce(function (previous, key) {
    return previous + parseFloat(pensionContributions[key] || 0);
}, 0);
```

Also could userInput be renamed since I don't get the meaning of the data from that variable name. Maybe `pensionContributions`?